gizmo.py

```diff
@@ -0,0 +1,37 @@
+def hello(name,country="Finland"):
+    print("Hello {0}, how are things in {1}?".format(str(name),str(country)))
+
+def spell(input_str):
+    result=input_str[0]
+    for i in input_str[1:]:
+        result+="."+i
+        
+    return print(result)
+
+def relative_path(subject_identifiers,relpath='./subjects/moc_recording'):
+    names=[]
+    relpath='./subjects/mock_recording'
+    for i in subject_identifiers:
+        res="{0}_{1}.rec".format(relpath,i) #new style
+        #res=relpath+"_"+i+".rec" #old style, not good
+        names.append(res)
```

Comment:
alternatively, you can use f-strings:

`res = f'./subjects/mock_recording_{i}.roc'`